lib/new_note_screen/how_was_your_day_widget.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+class HowWasYourDayWidget extends StatefulWidget {
+  const HowWasYourDayWidget({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<HowWasYourDayWidget> createState() => _HowWasYourDayWidgetState();
+}
+
+class _HowWasYourDayWidgetState extends State<HowWasYourDayWidget> {
+  bool value1 = true;
```

Comment:
Refactor to approach with Enum (e.g BottomNavigation implementation, but replace int index with enum index).
Move this logic to ChangeNotifier

Reply:
done